### ISSUE TITLE
All synced files are included in iCloud backup

### ISSUE THREAD

**Issue by sjpickup** [MobiusSync/MobiusSync#12]
All files synced by Möbius Sync are included in iCloud backup. Whilst we encourage backup external to Syncthing, users should be given the ability to disable this as they may prefer to backup from one of their other devices.

We could add a per-folder setting to include/exclude from iCloud backups.

Meanwhile, to exclude Möbius Sync completely from iCloud backups:

1. Go to Settings > [your name] > iCloud.
2. Tap Manage Storage > Backups.
3. Tap the name of the device you're using.
4. Under Choose Data to Back Up, turn off **Möbius Sync**.
5. Choose Turn Off & Delete.

For further info, see https://support.apple.com/en-gb/HT204247#backups

This should be explained in the FAQ.

**Comment by eecsfuture** [comment]
Hi, is there any way to let the sync folder path in iCloud drive?

By now, the Möbius Sync folder can only show up in "On My iPhone". 

And I can't find the button to include/exclude Möbius Sync in iCloud backups settings. 

My device is iPhone11, iOS 14.4

**Comment by adiabatic** [comment]
This seems to also be the case for local backup. I'm running my first post-Möbius-Sync-install backup to my iMac and it's taking an extra long time — about as much time as you'd expect to transfer 57 GB of files over a sync cable.

**Comment by ghost** [comment]
Is there any way to ignore folders in iTunes (Windows) backups?

**Comment by fschulze** [comment]
It would be amazing if this could be toggled per folder or at least globally also for local backups. I already backup my syncthing stuff and this just causes quite a lot of duplication.

**Comment by tve** [comment]
I do not believe that "exclude Möbius Sync completely from iCloud backups" is possible. iPad running iOS 18.6.2 when I look at the list of "Saved to iCloud" then Mobius Sync does not show up so I can't deselect it.
